csi/server/deploy/kubernetes/csi-configmap-opensdsplugin.yaml

```diff
@@ -7,3 +7,11 @@ metadata:
   name: csi-configmap-opensdsplugin
 data:
   opensdsendpoint: http://127.0.0.1:50040
+  opensdsauthstrategy: noauth
+  osauthurl: http://127.0.0.1/identity
```

Comment:
I think the osauthurl should also be updated since csi plugin is deployed in container.